packages/block-editor/src/components/media-upload/index.native.js

```diff
@@ -19,9 +16,9 @@ import { Picker } from '@wordpress/components';
 export const MEDIA_TYPE_IMAGE = 'image';
 export const MEDIA_TYPE_VIDEO = 'video';
 
-export const MEDIA_UPLOAD_BOTTOM_SHEET_VALUE_CHOOSE_FROM_DEVICE = 'choose_from_device';
-export const MEDIA_UPLOAD_BOTTOM_SHEET_VALUE_TAKE_MEDIA = 'take_media';
-export const MEDIA_UPLOAD_BOTTOM_SHEET_VALUE_WORD_PRESS_LIBRARY = 'wordpress_media_library';
+export const MEDIA_UPLOAD_BOTTOM_SHEET_VALUE_CHOOSE_FROM_DEVICE = 'DEVICE_MEDIA_LIBRARY';
+export const MEDIA_UPLOAD_BOTTOM_SHEET_VALUE_TAKE_MEDIA = 'DEVICE_CAMERA';
+export const MEDIA_UPLOAD_BOTTOM_SHEET_VALUE_WORD_PRESS_LIBRARY = 'SITE_MEDIA_LIBRARY';
```

Comment:
Maybe is better to define this in the bridge file, since this has to match the definition on the native side of the bridge too.

Reply:
Agree, these constants should be in the bridge file that has the method definition, so in the future if other component needs to use the picker it can directly from there.